src/ApiNavigationList.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+	<NcAppNavigationList :aria-labelledby="arialabel">
+		<NcAppNavigationItem v-for="app in apps"
+			:key="app.id"
+			allow-collapse
+			:name="app.name"
+			:open="isSelectedChild($route.params.appid, app)"
+			:to="{path: `/view/${app.id}`}">
+			<template #icon>
+				<AppIcon v-if="app.preview"
+					:href="app.preview" />
+				<ApiIcon v-else />
+			</template>
+			<NcAppNavigationItem v-for="child in childApis(app)"
+				:key="child.id"
+				:name="child.name"
+				:to="{path: `/view/${child.id}`}" />
+		</NcAppNavigationItem>
```

Comment:
IMO this should show the openapi-full.json by default if it exists and list  the openapi.json in the drop down.

Reply:
You're maybe right, but it generates some U/X questions (and a debate ;) ) : 
- what becomes the "short" api name ? -> you'll have "Provisioning API", then "provisioning_api" in the dropdown ?
- Shoud we remove the -full from the dropdown list ?
- Should be complete the name as "not full" for the simple api ?

My question is rather : why people generating openapi.json think it could be usefull to have a reduce set of api (openapi.json), some others sets, and finally a -full ? Is it a requirement from NC ?
If the purpose is "We thought the basic openapi.json is the one you'll use in 99% of cases, and the other are separated because they're very rare", then we should keep the simple way.
And finally, whatever they had in mind, the openapi.json is probably created to be the first to open and read ??

As a conclusion, i suggest to "keep it simple", and wait for at least on person requiring to do something with the "-full" ! ;)

Reply:
You're probably right.

I think we should change this at some point in openapi-extractor to generate openapi-clients.json instead of openapi.json and openapi.json instead of openapi-full.json, but we don't need to have to do anything here for now.